seatunnel-core/seatunnel-core-sql/src/main/java/org/apache/seatunnel/core/sql/splitter/SqlStatementSplitter.java

```diff
@@ -50,31 +54,32 @@
         List<String> statements = new ArrayList<>();
         List<String> buffer = new ArrayList<>();
 
-        for (String line : content.split("\n")) {
+        for (String line : content.split(LINE_SEPARATOR)) {
             if (isEndOfStatement(line)) {
                 buffer.add(line);
-                statements.add(normalizeLine(buffer));
+                statements.addAll(normalizeLine(buffer));
                 buffer.clear();
             } else {
                 buffer.add(line);
             }
         }
         if (!buffer.isEmpty()) {
-            statements.add(normalizeLine(buffer));
+            statements.addAll(normalizeLine(buffer));
         }
         return statements;
     }
 
     /**
      * Remove comment lines.
      */
-    private static String normalizeLine(List<String> buffer) {
-        return buffer.stream()
-                .map(statementLine -> statementLine.replaceAll(BEGINNING_COMMENT_MASK, ""))
-                .collect(Collectors.joining("\n"));
+    private static List<String> normalizeLine(List<String> buffer) {
```

Comment:
hi, have a question why change `String` to `List<String>`?

Reply:
please see test case, normalizeStatementsWithMultiSqls, if mult sql in one lines, it will wrong to return one string

Reply:
when `--` is the prefix of the statement, the whole statement is comment, we shouldn't split it, please revert this.

![image](https://user-images.githubusercontent.com/95013770/148172158-f5d46317-5e83-4d65-a7a2-6d2f7b06ae7d.png)

Reply:
hi, we already handle the logic in `SqlStatementSplitter#splitContent`, it will return the statements.

Reply:
sorry, don't got it

Reply:
hi, if sql statements like following, it will return wrong statements.
```
String sqlContent = "--test is a comment \n select * from dual; select now(); select * from logs where log_content like ';'";
```

Reply:
got it, reverted